deep_qa/data/token_indexers/token_indexer.py

```diff
@@ -75,7 +74,6 @@ def pad_token_sequence(self,
         """
         raise NotImplementedError
 
-    @staticmethod
-    def from_params(params: Params):
-        choice = params.pop_choice('type', list(token_indexers.keys()), default_to_first_choice=True)
-        return token_indexers[choice].from_params(params)
```

Comment:
We can't have this here because it creates circular imports - `token_indexers` are imported from `__init__` but haven't been defined yet. We also can't import the classes directly because they inherit from this class.